index.js

```diff
@@ -0,0 +1,19 @@
+const parser = require('tap_parser')
+const duplex = require('duplexer')
+const {obj: through} = require('throo')
```

Comment:
this is too new syntax, it should work with node v4 at least. also, i'd prefer to just use `through2`, it's a very common dependency.